cmd/bucket-notification-utils_test.go

```diff
@@ -337,11 +332,6 @@ func TestUnmarshalSQSARN(t *testing.T) {
 			queueARN: "",
 			Type:     "",
 		},
-		// Invalid region 'us-west-1' in queue arn.
-		{
-			queueARN: "arn:minio:sqs:us-west-1:1:redis",
```

Comment:
this test gets covered by other test cases which uses `us-east-1` which is not treated specially anymore